features/networking/multicast.feature

```diff
@@ -159,13 +159,14 @@ Feature: testing multicast scenarios
     And evaluation of `pod(2).ip` is stored in the :proj1pod3ip clipboard
     And evaluation of `pod(2).name` is stored in the :proj1pod3 clipboard
     # Enable multicast in proj1
+    Given I store multicast keyval according to the networktype in the :keyval clipboard
+    Given I store multicast namespace according to the networktype in the :namespace clipboard
     When I run the :annotate admin command with:
-      | resource     | netnamespace                                             |
-      | resourcename | <%= cb.proj1 %>                                          |
-      | overwrite    | true                                                     |
-      | keyval       | netnamespace.network.openshift.io/multicast-enabled=true |
+      | resource     | <%= cb.namespace %>   |
+      | resourcename | <%= cb.proj1 %>       |
+      | keyval       | <%= cb.keyval %>      |
```

Comment:
Instead of having a step to get annotation, store in clipboard, then apply, why not write a step
`I enable multicast for the #{QUOTED} project`? Will simplify a lot the scenario.

Reply:
@akostadinov  Good suggestion, create Given /^I enable multicast for the "(.+?)" namespace now.
New log from testing ovn cluster:
https://privatebin-it-iso.int.open.paas.redhat.com/?b29383740e18b36d#c2jFDsUgg39zYNyBJaRADG8YUYYCtZcYDXSuRJIPnTI=

New log from testing sdn cluster:
https://privatebin-it-iso.int.open.paas.redhat.com/?538718d77b3cbbdc#23kXoYOHTbWMQ2LkQyzMROjkJFq/uxcJ754Px7qZf30=